webapi/formatting.go

```diff
@@ -26,3 +32,24 @@ func blockURL(blockExplorerURL string) func(int64) string {
 func dateTime(t int64) string {
 	return time.Unix(t, 0).Format("2 Jan 2006 15:04:05 MST")
 }
+
+func stripWss(input string) string {
+	input = strings.ReplaceAll(input, "wss://", "")
+	input = strings.ReplaceAll(input, "/ws", "")
+	return input
+}
+
+func indentJSON(input string) template.HTML {
+	var indented bytes.Buffer
+	err := json.Indent(&indented, []byte(input), "<br/>", "&nbsp;&nbsp;&nbsp;&nbsp;")
+	if err != nil {
+		log.Errorf("Failed to indent JSON: %w", err)
```

Comment:
Avoid capitalizing the error string here.

Reply:
This is just a logging statement which will be printed in isolation, thus the caps. This matches the rest of the log statements in the project. Error strings on the other hand are lower-cased because they are usually printed with some proceeding context.